scss/_roster.scss

```diff
@@ -233,7 +233,7 @@
     }
 }
 
-#jsxc_roster > {
+#jsxc_roster > & {
     &.jsxc_bottom {
```

Comment:
With node-sass@3.3.2 it works like a charm. Forcing it (#246) should fix the issue.